assets/js/components/ImageWithPlaceholder.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { createUseStyles } from 'react-jss';
+import PropTypes from 'prop-types';
+
+const useStyle = createUseStyles({
+  placeholder: {
+    height: '100%',
+    width: '100%',
+    objectFit: 'contain',
+  },
+  img: {
+    extend: 'placeholder',
+    display: (loaded) => (loaded ? 'block' : 'none'),
+  },
+});
+
+export const ImageWithPlaceholder = React.memo(({ img, placeholder }) => {
+  const [loaded, setLoaded] = useState(false);
+  const classes = useStyle(loaded);
+
+  return (
+    <>
+      {!loaded && <img className={classes.placeholder} src={placeholder} alt={placeholder} />}
+      {<img className={classes.img} src={img} alt={img} onLoad={() => setLoaded(true)} />}
```

Comment:
I'd leave the `alt`s empty because they don't add more information about the image.

Reply:
As you suggest, for now, I've added it as a parameter.